com.unity.netcode.gameobjects/Components/HalfVector4.cs

```diff
@@ -0,0 +1,68 @@
+using UnityEngine;
+
+namespace Unity.Netcode.Components
+{
+    internal struct HalfVector4 : INetworkSerializable
+    {
+        public ushort X;
```

Comment:
I'm curious now if there's a reason you used ushort[] for HalfVector3, but just used four values for HalfVector4.